src/contrib/cluster/Akka.DistributedData/Replicator.cs

```diff
@@ -441,8 +449,6 @@ private void Load()
             var startTime = DateTime.UtcNow;
             var count = 0;
 
-            NormalReceive();
```

Comment:
Instead of letting the node to process everything while loading, we will stash messages instead. The only things that are processed normally during load are cluster domain events.